plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -442,13 +446,13 @@ private boolean validateRequiredGroups(MissingRequiredProperties validationFailu
     if (missingGroupEntry != null) {
       StringBuilder errorBuilder = new StringBuilder("Missing value for group ");
       errorBuilder.append(missingGroupEntry.getKey());
-      errorBuilder.append(". Properties satisfying group requirement are ");
+      errorBuilder.append(". Properties satisfying group requirement are "); //$NON-NLS-1$
       Set<String> groupMembers = new HashSet<>();
       for (PipelineOptionsProperty missingProperty : missingGroupEntry.getValue()) {
         groupMembers.add(missingProperty.getName());
       }
       errorBuilder.append(MISSING_GROUP_MEMBER_JOINER.join(groupMembers));
-      errorBuilder.append(".");
+      errorBuilder.append("."); //$NON-NLS-1$
       setErrorMessage(errorBuilder.toString());
```

Comment:
There's no need for ICU's MessageFormat.
```
// "Missing value for group {0}. Properties satisfying group requirement are {1}."
MessageFormat.format("missing.property.value.for.group",
   missingGroupEntry.getKey(),
   ListFormatter.getInstance().format(groupMembers));
```
Produces:
<img width="765" alt="screen shot 2017-08-10 at 2 45 32 pm" src="https://user-images.githubusercontent.com/202851/29186471-9b8c1342-7dda-11e7-8c23-5d6fce185f1e.PNG">

That said, I can't actually excercise this codepath with the DirectRunner and DataflowRunners as they don't seem to specify any required grouped properties.  I had to hack up the code instead.